tests/api.c

```diff
@@ -14731,6 +14736,143 @@ static void test_wc_PKCS7_EncodeEncryptedData (void)
 } /* END test_wc_PKCS7_EncodeEncryptedData() */
 
 
+/* Testing wc_SignatureGetSize() */
+static int test_wc_SignatureGetSize(void) {
+
+    int ret = 0; 
+    enum wc_SignatureType sig_type;
+    word32 key_len;
+    ecc_key ecc; 
+    RsaKey rsa_key;
```

Comment:
RSA-specific API that comes from <wolfssl/wolfcrypt/rsa.h> should be protected with an "#ifndef NO_RSA". This will compile out those sections when the user has disabled RSA.